src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -759,6 +764,10 @@
 examples:
   - name: Validate whether a template is valid at management group.
     text: az deployment mg validate --management-group-id testmg --location WestUS --template-file {template-file}
+  - name: Validate whether a template is valid at management group. (autogenerated)
+    text: |
+        az deployment mg validate --location WestUS --management-group-id testmg --name mydeployment --parameters MyValue=This MyArray=@array.json --template-file azuredeploy.json
```

Comment:
> --parameters MyValue=This MyArray=@array.json

The sample of this parameter doesn't look very clear~

Reply:
What then? Could you apply suggestions as well?

Reply:
How about `@myparameters.json` ?

Reply:
Will this value `MyValue=This MyArray=@array.json` of --parameters be valid? 
If so, is it encouraged to demo more flexible usage this argument?

Reply:
```suggestion
        az deployment mg validate --location WestUS --management-group-id testmg --name mydeployment --parameters @myparameters.json --template-file azuredeploy.json
```